Dangerfile

```diff
@@ -0,0 +1,45 @@
+# set the number of lines that must be changed before this classifies as a 'Big PR'
+@SDM_DANGER_BIG_PR_LINES = 50
```

Comment:
what does the SDM prefix mean in this context?

Reply:
I don't know :(
I thought it's just a variable name, copied from here https://github.com/samdmarshall/danger/blob/master/Dangerfile

Reply:
Honestly, I don't know. Just saw that naming many times in other repos.
https://github.com/search?q=SDM_DANGER_BIG_PR_LINES&type=Code

Reply:
Probably more Dangerfiles without it :) It doesn't make much sense, I'd remove it

Reply:
Removed, thanks